### PR TITLE
Dark mode for ENHO and SFPF to enable RFC pull

### PR DESCRIPTION
**Pull request by filak-sap**
I pull our repositories to newly created systems via RFC helpers and I had to prepare these to commits to make it working.

The commits have been deployed in all our systems for more then a month and I got not angry emails yet - so I bet those tens of users didn't run into any issue with them.

### CODE REVIEW

File: src/objects/zcl_abapgit_object_sfpf.clas.abap [hunk 1]
```diff
@@ -194,15 +194,19 @@ CLASS ZCL_ABAPGIT_OBJECT_SFPF IMPLEMENTATION.
...
 
 
     lo_wb_form ?= load( ).
 
     lv_name = ms_item-obj_name.
+    lv_corrnum = zcl_abapgit_default_transport=>get_instance( )->get( )-ordernum.
 
     TRY.
-        lo_wb_form->delete( lv_name ).
+        lo_wb_form->delete( i_name = lv_name
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_sfpf.clas.abap:207]: see below

File: src/objects/zcl_abapgit_object_sfpf.clas.abap [hunk 2]
```diff
@@ -232,12 +239,16 @@ CLASS ZCL_ABAPGIT_OBJECT_SFPF IMPLEMENTATION.
         ENDIF.
 
         IF zif_abapgit_object~exists( ) = abap_true.
-          cl_fp_wb_form=>delete( lv_name ).
+          cl_fp_wb_form=>delete( i_name = lv_name
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_sfpf.clas.abap:242]: 702

File: src/objects/zcl_abapgit_object_sfpf.clas.abap [hunk 3]
```diff
@@ -232,12 +239,16 @@ CLASS ZCL_ABAPGIT_OBJECT_SFPF IMPLEMENTATION.
...
+                                 i_ordernum = lv_corrnum
+                                 i_dark = 'X' ).
         ENDIF.
 
         tadir_insert( iv_package ).
         li_wb_object = cl_fp_wb_form=>create( i_name = lv_name
-                                              i_form = li_form ).
+                                              i_form = li_form
```
Comment (mbtools) [inline src/objects/zcl_abapgit_object_sfpf.clas.abap:249]: 702

**Review by mbtools — CHANGES_REQUESTED** [review]
702 compatibility

**Comment by mbtools** [comment]
702: no dark_mode for sfpf

image

image

**Comment by filak-sap** [comment]
Alright, keep it as a down stream patch. Thank you!

**Comment by mbtools** [comment]
Don't close. It should be included . Just call the methods twice like this
https://github.com/abapGit/abapGit/blob/main/src/objects/zcl_abapgit_object_devc.clas.abap#L194

**Pull request closed without merging**